metrics/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
+from django.db import models
+
+from topics.models import Article, Topic
+
+
+class Metric(models.Model):
+    article = models.ForeignKey(Article)
+    topic = models.ForeignKey(Topic)
+    owner = models.ForeignKey(User)
+
+    location = models.IntegerField()
```

Comment:
Yeah agreed; It's just that purely a character index of a massive pile of HTML simply won't work as the length of the source could constantly change. If we get the ID of the container then this is way more robust as we can find the container again even though the site has completely changed (unless the ID of the container changes as well). We should think about this, but sure we can keep this for now.